src/include/miopen/prelu/problem_description.hpp

```diff
@@ -53,6 +53,10 @@ struct BackwardProblemDescription : ProblemDescriptionBase
     {
         IsSameType();
         IsRightLength();
+        if(!weightDesc.IsPacked())
```

Comment:
Please refer to https://github.com/moreh-dev/MIOpen/pull/12#discussion_r1597507827 and change packed to contiguous.

Reply:
In this operation, `weight` and `dweight` are 1D tensors. They only can be contiguous once their `stride[0] != 1`, which mean `IsPacked()` return false. Therefore, I think implement a contiguous check function is excessive.